docs/api-methods.rst

```diff
@@ -0,0 +1,221 @@
+.. Note, this was autogenerated, all changes will vanish...
+
+Available JSON-RPC methods
+==========================
+
+
+version
+-------
+
+Get the version of conferatur
+
+:return str: Conferatur version
+
+normalization.list
+------------------
+
+Get a list of available core normalizers
+
+:return object: With key being the normalizer name, and value its description
+
+normalization.alphanumeric
+--------------------------
+
+Simple alphanumeric filter
```

Comment:
No use case really, as I'm now tentatively starting to implement the "tokenization" classes, this would be pretty much replaced by that (i.e. tokenization would split words on spaces). Some specific normalization classes that exist now will be quickly removed and replaced as this project goes on. Some of them currently exist as they were the first test normalization methods. I like to keep them around as first reference and simple implementation examples until next version.